app/models/server/raw/Rooms.js

```diff
@@ -397,4 +397,22 @@ export class RoomsRaw extends BaseRaw {
 	findOneByNameOrFname(name, options = {}) {
 		return this.col.findOne({ $or: [{ name }, { fname: name }] }, options);
 	}
+
+	allRoomSourcesCount() {
+		return this.col.aggregate([
+			{
+				$match: {
+					source: {
+						$exists: true,
+					},
```

Comment:
```suggestion
					},
					t: 'l',
```

Reply:
Although I do understand that source property will only exist on live-chat conversations, I'm still suggesting this to be extra cautious 😁